tests/conf/nccmsentinel2.yaml

```
@@ -0,0 +1,19 @@
+model:
+  class_path: SemanticSegmentationTask
+  init_args:
+    loss: "ce"
+    model: "deeplabv3+"
+    backbone: "resnet34"
+    in_channels: 4
+    num_classes: 14
+    num_filters: 1
+    ignore_index: null
```

```suggestion
    num_classes: 4
    num_filters: 1
    ignore_index: 4
```
The nodata class is 4, which should be ignored during training. The actually classes are class 0-3.